object_tracker/object_tracker.go

```diff
@@ -70,10 +85,13 @@ func newTracker(ctx context.Context, deps resource.Dependencies, conf resource.C
 		classCounter: make(map[string]int),
 		tracks:       make(map[string][]objdet.Detection),
 		properties: vision.Properties{
-			ClassificationSupported: false,
+			ClassificationSupported: true,
 			DetectionSupported:      true,
 			ObjectPCDsSupported:     false,
 		},
+		ac: allClassifications{
+			class: []string{},
```

Comment:
While the class string has the timestamp in the name, could you also save the time-tamp as another field in the entry? You could even extract the timestamp from the label string to store.

Reply:
done it's a `map[string]interface{}` now